lib/ae_mdw/db/name.ex

```diff
@@ -104,7 +104,7 @@ defmodule AeMdw.Db.Name do
     m_name_exp = Model.expiration(index: {expire, plain_name})
     m_owner = Model.owner(index: {owner, plain_name})
 
-    ensure_no_exsiting_active_name_expiration(plain_name)
+    clean_previous_expirations(previous || [])
```

Comment:
Isn't `previous` here the result of an `{:ok, inactive_name}` tuple (not a list)?

Reply:
It returns `nil` when there is not record (with `map_one_nil`).

Reply:
I was referring to the other case, what happens when it's not `nil`? Is it a list?

Reply:
correct and exist only when name is reactivated (doesn't cover extensions) so better to remove it and rely on:
https://github.com/aeternity/ae_mdw/blob/72c500109cfb15afb11a17fdaafe7ef920d5a5a5/lib/ae_mdw/db/name.ex#L68